### PR TITLE
implemented TODO changes from testutil.py

### PR DESCRIPTION
**Pull request by shrishinde**
@whummer 
Please take a look.

### CODE REVIEW

**Comment by shrishinde** [comment]
Now running all tests using `make test`. Will figure it out locally.

**Comment by shrishinde** [comment]
python 3 did not like `"w"` changed it to `'w'`

File: localstack/utils/testutil.py [hunk 1]
```diff
@@ -50,52 +52,61 @@ def create_dynamodb_table(table_name, partition_key, env=None, stream_view_type=
...
             root_folder = LOCALSTACK_ROOT_FOLDER
-            target_dir = '%s/%s/' % (tmp_dir, lib)
+            target_dir = os.path.join(tmp_dir, lib)
             mkdir(target_dir)
         for path in paths:
-            file_to_copy = '%s/%s' % (root_folder, path)
+            file_to_copy = os.path.join(root_folder, path)
             for file_path in glob.glob(file_to_copy):
-                run('cp -r %s %s/' % (file_path, target_dir))
+                name = os.path.join(target_dir, file_path.split('/')[-1])
```
Comment (whummer) [inline localstack/utils/testutil.py]: Better replace `'/'` with `os.path.sep` here.

File: localstack/utils/testutil.py [hunk 2]
```diff
@@ -50,52 +52,61 @@ def create_dynamodb_table(table_name, partition_key, env=None, stream_view_type=
...
+                    shutil.copyfile(file_path, name)
 
     # create zip file
-    return create_zip_file(tmp_dir, get_content=True)
+    return create_zip_file(tmp_dir, get_content=get_content)
 
 
-# TODO: Refactor this method and use built-in file operations instead of shell commands
-def create_zip_file(file_path, include='*', get_content=False):
+def create_zip_file(file_path, get_content=False):
```
Comment (whummer) [inline localstack/utils/testutil.py]: Looks like `create_zip_file(..)` is only used from this file, so it should be safe to remove the `include` parameter here. 👍

**Comment by whummer** [comment]
@shrishinde Thanks for fixing this `TODO` - looks great. I only left one nitpick comment (replace '/' with `os.path.sep`). Thanks

**Pull request merged**